WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -122,6 +122,37 @@
         });
       </script>
 
+
+      <% if @competition_admin_view %>
```

Comment:
By "set", do you mean "before they click the save button"? If we want to do something realtime and fancy, we'll need to use javascript. It's doable, but gets a little hacky. This is not the sort of thing javascript is very good at. @timhabermaas, do you have any suggestions here?

Why does nothing show up if there is no date? Would other unscheduled competitions show up if there were any?